packages/@aws-cdk/aws-rds/README.md

```diff
@@ -25,7 +25,7 @@ your instances will be launched privately or publicly:
 const cluster = new DatabaseCluster(this, 'Database', {
     engine: DatabaseClusterEngine.AURORA,
     masterUser: {
-        username: 'admin'
+        username: 'clusteradmin'
```

Comment:
should we add a validation that will help people avoid this?

Reply:
I'm not sure about this. According to the docs, the master username blacklist varies by database engine: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html

To properly validate, the CDK would have to be aware of all reserved words for all available engines. The list of reserved words might be quite large.

Reply:
ok fair enough. Perhaps we can add this link to the docs of `username` and recommend that users look at that?